tests/interfaces/default_qubit_2_integration/test_tensorflow_autograph_qnode_shot_vector_default_qubit_2.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Integration tests for using the TF interface with shot vectors and with a QNode"""
-# pylint: disable=too-many-arguments,too-few-public-methods,unexpected-keyword-arg
+# pylint: disable=too-many-arguments,too-few-public-methods,unexpected-keyword-arg,redefined-outer-name
```

Comment:
I'm not fond of this warning. I'd prefer if you just removed the reused arg name

Reply:
But that's the fixture's name, I dunno why it is complaining about it